test/models/changeset/pull_request_test.rb

```diff
@@ -29,6 +30,19 @@
     end
   end
 
+  describe ".changeset_from_webhook" do
+    before do
+      Rails.cache.clear
```

Comment:
Leftover from when the `changeset_from_webhook` was making a github api call. thanks.